core/templates/pages/topic-editor-page/topic-editor-page.module.ts

```diff
@@ -22,9 +22,7 @@ import 'zone.js';
 import { Component, NgModule, StaticProvider } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
 import { downgradeComponent } from '@angular/upgrade/static';
-import { HttpClientModule } from '@angular/common/http';
-import { HTTP_INTERCEPTORS } from '@angular/common/http';
-import { RequestInterceptor } from 'services/request-interceptor.service';
+import { HttpClientModule, HTTP_INTERCEPTORS } from '@angular/common/http';
```

Comment:
Why are we re-ordering the imports in this file?

Reply:
Oh I missed it. Must have happened while resolving merge conflicts. Fixed it and thanks!